src/compile.coffee

```diff
@@ -148,7 +147,11 @@ compileOriginExampleName = (mediaType, httpResponseElement, exampleNo) ->
   else
     statusCode = httpResponseElement.statusCode.toValue()
     headers = compileHeaders(httpResponseElement.headers)
-    contentType = caseless(headers).get('content-type')?.value
+
+    contentType = headers
+      .filter((header) -> header.name.toLowerCase() is 'content-type')
+      .map((header) -> header.value)
+      .join(', ')
```

Comment:
I do not believe it is possible to have multiple `Content-Type` headers. A brief look at RFC7231 would indicate this too:

https://tools.ietf.org/html/rfc7231#section-3.1.1.5
https://tools.ietf.org/html/rfc7231#section-3.1.1.1

Reply:
Also, not sure what that would semantically mean, as a body would only be a single content-type?

Reply:
I do not believe in multiple content type headers as well. I think the `join` is just confusing, I should change it. The array _should_ contain just one item, so this is equivalent to `[0]`.